README.md

```diff
@@ -1,16 +1,10 @@
-# This is the v3 version of [cms-sanity](https://github.com/vercel/next.js/tree/canary/examples/cms-sanity)
+# Sanity.io and Next.js
 
-Use this deploy link while jumpstart is in development (it uses the staging integration with the new flow): [Deploy (Staging)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fsanity-io%2Fvercel-next.js-examples-cms-sanity-studio-v3&repository-name=blog-nextjs-sanity&project-name=blog-nextjs-sanity&demo-title=Blog%20using%20Next.js%20%26%20Sanity&demo-description=On-demand%20ISR%2C%20sub-second%20as-you-type%20previews&demo-url=https%3A%2F%2Fnextjs-blog.sanity.build%2F&demo-image=https%3A%2F%2Fuser-images.githubusercontent.com%2F110497645%2F182727236-75c02b1b-faed-4ae2-99ce-baa089f7f363.png&integration-ids=oac_Cf6NwaRiVHouq8BfQAR802qo)
+![main](./assets/readme/main.png)
 
-> If you're running jumpstart on http://localhost:3001 then use this link: [Deploy (Development)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fsanity-io%2Fvercel-next.js-examples-cms-sanity-studio-v3&repository-name=blog-nextjs-sanity&project-name=blog-nextjs-sanity&demo-title=Blog%20using%20Next.js%20%26%20Sanity&demo-description=On-demand%20ISR%2C%20sub-second%20as-you-type%20previews&demo-url=https%3A%2F%2Fnextjs-blog.sanity.build%2F&demo-image=https%3A%2F%2Fuser-images.githubusercontent.com%2F110497645%2F182727236-75c02b1b-faed-4ae2-99ce-baa089f7f363.png&integration-ids=oac_5lzYFz8Fc3nFPbTWXb73BDwa&external-id=nextjs)
+Sanity.io is the platform for structured content. With Sanity.io you can manage your text, images, and other media with APIs and use the open-source single page application Sanity Studio to quickly set up an editing environment that you can customize.
 
----
-
-# A statically generated blog example using Next.js and Sanity
-
-[![Deploy with Vercel](https://vercel.com/button)][vercel-deploy]
-
-This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Sanity](https://www.sanity.io/) as the data source.
+This is a statically generated blog that uses Next.js and Sanity. It showcases Next.js's Static Generation feature using Sanity as the data source.
```

Comment:
This could also be more brief e.g. "This is a statically generated blog using Next.js and Sanity. It uses Next.js's Static Generation and Sanity as **a** data source."

Reply:
Along the lines of:

Want to try [Sanity.io](http://sanity.io/) with Next.js? The Sanity editor is Just React™ so for this starter we're setting it up for you on /studio . It comes with real-time collaboration and instant as-you-type previews. We also, further down, show you how to configure ISR (incremental static rebuidling) for super porformant sites that also can rebuild instantly.

Reply:
Updated the content (thanks for the suggestion Even): https://github.com/sanity-io/vercel-next.js-examples-cms-sanity-studio-v3/pull/25#discussion_r999589934